{"changes": {"state": {"new": "open", "old": "invalid"}}, "links": {"self": {"href": "data/repositories/philipstarkey/qtutils/issues/6/changes/46892782.json"}, "html": {"href": "#!/philipstarkey/qtutils/issues/6#comment-46892782"}}, "issue": {"links": {"self": {"href": "data/repositories/philipstarkey/qtutils/issues/6.json"}}, "type": "issue", "id": 6, "repository": {"links": {"self": {"href": "data/repositories/philipstarkey/qtutils.json"}, "html": {"href": "#!/philipstarkey/qtutils"}, "avatar": {"href": "data/bytebucket.org/ravatar/{eb85b0e6-11dc-449a-96ec-21935a03d721}ts=python"}}, "type": "repository", "name": "qtutils", "full_name": "philipstarkey/qtutils", "uuid": "{eb85b0e6-11dc-449a-96ec-21935a03d721}"}, "title": "Installed wants qt4 and pyside"}, "created_on": "2018-07-24T00:45:22.197667+00:00", "user": {"display_name": "Philip Starkey", "uuid": "{0147401a-13ed-4e39-a0d0-63108c18738b}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B0147401a-13ed-4e39-a0d0-63108c18738b%7D"}, "html": {"href": "https://bitbucket.org/%7B0147401a-13ed-4e39-a0d0-63108c18738b%7D/"}, "avatar": {"href": "data/secure.gravatar.com/avatar/fa0698c306b3470c00717c146b5296e9d=httpsavatar-management--avatars.us-west-2.prod.public.atl-paas.netinitialsPS-0.png"}}, "nickname": "philipstarkey", "type": "user", "account_id": "557058:2f99420c-1dbd-4837-952c-82c421b8fbdd"}, "message": {"raw": "Reopening.\n\nThe bug is that, while there is an error to catch the rcc executable not existing (which notifies about the NO_XXX options), this doesn't happen because it first tries to import the library (see [here](#!/philipstarkey/qtutils/src/d087c7e6fc80c43463bbb88f3a77a90e41f46e81/qtutils/icons/_build.py#lines-57)). I have no idea why it's importing the library because it doesn't seem to ever use it....but it does the same thing for all 4 of the python Qt wrappers.", "markup": "markdown", "html": "<p>Reopening.</p>\n<p>The bug is that, while there is an error to catch the rcc executable not existing (which notifies about the NO_XXX options), this doesn't happen because it first tries to import the library (see <a data-is-external-link=\"true\" href=\"#!/philipstarkey/qtutils/src/d087c7e6fc80c43463bbb88f3a77a90e41f46e81/qtutils/icons/_build.py#lines-57\" rel=\"nofollow\">here</a>). I have no idea why it's importing the library because it doesn't seem to ever use it....but it does the same thing for all 4 of the python Qt wrappers.</p>", "type": "rendered"}, "type": "issue_change", "id": 46892782}